src/molgenis/capice/vep/template_exon_intron.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
+
+import pandas as pd
+
+from molgenis.capice.vep.template import Template
+
+
+class TemplateExonIntron(Template):
+    def __init__(self, name='Template', usable=False):
+        super(TemplateExonIntron, self).__init__(
+            name=name,
+            usable=usable
+        )
+
+    @property
+    @abstractmethod
+    def columns(self):
+        return [None, None]
```

Comment:
Consider removing it and adding some pydoc to class itself explaining subclass expectations (as the code itself doesn't have an actual function).

Otherwise, at the very least add a third element (though I strongly suggest the solution above).